utils/src/main/scala/za/co/absa/enceladus/utils/implicits/StringImplicits.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.utils.implicits
+
+import java.security.InvalidParameterException
+
+import scala.annotation.tailrec
+
+object StringImplicits {
+  implicit class StringImprovements(string: String) {
```

Comment:
Just one small thing, the convention for implicits is to call things usually goes "RichX" or "WrappedX" or "xOps". ([Like it's described here](https://stackoverflow.com/questions/28061661/is-there-a-naming-convention-for-implicit-classes))